app/src/visionglass/java/com/igalia/wolvic/PlatformActivity.java

```diff
@@ -155,23 +159,33 @@ protected void onCreate(Bundle savedInstanceState) {
         initVisionGlass();
     }
 
+    private void onConnectionStateUpdated(PhoneUIViewModel.ConnectionState connectionState) {
+        Log.d(LOGTAG, "Connection state updated: " + connectionState);
+    }
```

Comment:
Since you're only using it for logging I think we can convert this into a lambda in the call site.

Reply:
Done.